app/views/devise/registrations/edit.html.haml

```diff
@@ -2,9 +2,14 @@
 = simple_form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put, class: "form-horizontal" }) do |f|
   = f.error_notification
   .form-inputs
-    = f.input :email, required: true, autofocus: true
-    = f.input :password, autocomplete: "off", hint: t("devise.registrations.leave_it_blank_if_dont_want_to_change"), required: false
-    = f.input :password_confirmation, required: false
-    = f.input :current_password, hint: t("devise.registrations.need_current_password_to_confirm"), required: true
-  .form-actions
-    = f.button :submit, t('common.update'), class: "btn btn-primary"
+    - if current_user.demo_user?
+      = f.input :email, required: false, autofocus: false, :input_html => { readonly: :true, :value => 'demo@homebugh.info' }
```

Comment:
```suggestion
      = f.input :email, required: false, autofocus: false, input_html: { readonly: true, value: current_user.email }
```

I would suggest to use a `current_user` email instead of a hard coded value. I've also applied the [Ruby 1.9 hash syntax](https://github.com/rubocop-hq/ruby-style-guide#hash-literals).

Reply:
Oooh sorry about the syntax mistakes, I will keep an eye on it when it comes to other Issues/PRs.
Good catch to use the current_user.email 💯